StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsLegalDetailsNotice.swift

```diff
@@ -8,12 +8,20 @@
 import Foundation
 
 struct FinancialConnectionsLegalDetailsNotice: Decodable {
+
+    let icon: FinancialConnectionsImage?
     let title: String
+    let subtitle: String?
     let body: Body
-    let learnMore: String
+    let disclaimer: String?
```

Comment:
[link to double check models](https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/text/data/legal_details_notice_v3.rb)